README.md

```diff
@@ -86,98 +86,6 @@ You can use a different compile with `dub --compiler ldc`.
 For a performance boost, add `-b release` to let the compiler perform additional
 optimizations, inlining, removal of bound checking and `assert` statements.
 
-### Rapid edit-run cycle without dub
```

Comment:
I have not time to fix the docs. No docs about rare case a better then wrong docs. You may want to create a wiki page